curriculum/challenges/english/07-scientific-computing-with-python/learn-classes-and-objects-by-building-a-sudoku-solver/6606d7bb9e4c6b574235159a.md

```diff
@@ -0,0 +1,69 @@
+---
+id: 6606d7bb9e4c6b574235159a
+title: Step 44
+challengeType: 20
+dashedName: step-44
+---
+
+# --description--
+
+The method returns `False` because `3` is already present in that square. Try with another square by changing the column index into `6`.
```

Comment:
```suggestion
The method returns `False` because `3` is already present in that square. Try with another square by changing the column index to `6`.
```